### PR TITLE
Feature/aps 1891 add upsert basket request

### PR DESCRIPTION
**Pull request by YuryStruchkou**

### CODE REVIEW

File: EncoreTickets.ConsoleTester/EncoreTickets.ConsoleTester.csproj [hunk 1]
```diff
@@ -18,6 +18,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Update="appsettings.real.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
```
Comment (TanyaEvtushenko) [inline EncoreTickets.ConsoleTester/EncoreTickets.ConsoleTester.csproj]: why is it duplicated here?

File: EncoreTickets.SDK/Basket/Models/Seat.cs [hunk 2]
```diff
@@ -4,9 +4,9 @@ public class Seat
     {
         public string aggregateReference { get; internal set; }
 
-        public string blockId { get; internal set; }
+        public string areaId { get; internal set; }
 
-        public string blockName { get; internal set; }
+        public string areaName { get; internal set; }
```
Comment (TanyaEvtushenko) [inline EncoreTickets.SDK/Basket/Models/Seat.cs]: Are you sure you need an internal modifier for model setters?

Comment (YuryStruchkou) [reply EncoreTickets.SDK/Basket/Models/Seat.cs]: It's currently used as a response model so I'm not sure why would we not make it readonly outside the SDK. Do you think we might need public access here?

Comment (TanyaEvtushenko) [reply EncoreTickets.SDK/Basket/Models/Seat.cs]: In the future, we can reuse models to add or update entities. I think this is an additional protection that is not very useful at the moment, but may be changed in the future for the purposes of unit testing and extensions of services

**Pull request merged**